### PR TITLE
Update the Release Post page with image details

### PR DESCRIPTION
**Pull request by iskyOS**
Handbook update.
Add details on how to capture images & video for the release post.

### CODE REVIEW

**Comment by iskyOS** [comment]
@bretthayes, I changed the release post blog guidance in the Handbook to show that images/videos should use the Figure tag rather than img (based on the blog post starter pack). Could you confirm that's correct?

File: content/departments/marketing/product-marketing/release_post_process.md [hunk 1]
```diff
@@ -39,18 +39,35 @@ You can tell when the release is cut by following along in #progress.
...
+For videos, keep these things in mind:
+
+- Try to keep videos shorter than 10 seconds.
+- Capture your content as videos (MP4 format) and not as GIFs.
+- Videos will be formatted in the release post to look and act like GIFs. They will autoplay on loop and be muted.
+
+### Embedding content
+
+For all images and videos added to the release post, include the `blog-image` class as follows, and be sure to add an `alt` and `title` description for hover and/or screen readers:
+
```
Comment (bretthayes) [inline content/departments/marketing/product-marketing/release_post_process.md]:
```suggestion
To render an [Image](https://about.sourcegraph.com/blog/starter-pack#figure) or [Video](https://about.sourcegraph.com/blog/starter-pack#video), please use the `Figure` and `Video` component from the [Blog Starter Pack](https://about.sourcegraph.com/blog/starter-pack).
```

File: content/departments/marketing/product-marketing/release_post_process.md [hunk 2]
```diff
@@ -39,18 +39,35 @@ You can tell when the release is cut by following along in #progress.
...
+````html
+<figure
   src="https://my-image.jpg"
   class="blog-image"
   alt="This description will be shown to people with assistive readers"
   title="This description will be shown on hover"
 />
```
Comment (bretthayes) [inline content/departments/marketing/product-marketing/release_post_process.md]:
```suggestion
Here is an example of rendering an image using the Figure component:

```mdx
<Figure
  src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
  alt="Alternative information or rendered when an image cannot be displayed"
  caption="An optional caption displayed under the figure"
  link="https://www.sourcegraph.com/my/optional/link"
  linkIcon={true}
/>
```

File: content/departments/marketing/product-marketing/release_post_process.md [hunk 3]
```diff
@@ -39,18 +39,35 @@ You can tell when the release is cut by following along in #progress.
...
-
-The `alt` and `title` values can be the same or not, up to you. In general, `title` is more appropriate for a general
-title, and `alt` is more appropriate to describe what you're seeing visually.
-```
+``` The `alt` and `title` values can be the same or not, up to you. In general, `title` is more appropriate for a
+general title, and `alt` is more appropriate to describe what you're seeing visually. For more information, see the
+[blog post start pack](https://handbook.sourcegraph.com/departments/marketing/blog/).
+````
```
Comment (bretthayes) [inline content/departments/marketing/product-marketing/release_post_process.md]:
```suggestion
The `alt` prop is automatically used for the `title` prop so all that's required is the alt prop. The title attribute specifies extra information about an element and is most often shown as a text tooltip when the mouse moves over the element. For more information on available props and documentation, see the [Blog Starter Pack](https://about.sourcegraph.com/blog/starter-pack) and the [Figure Component](https://about.sourcegraph.com/blog/starter-pack#figure).
```

**Review by bretthayes — COMMENTED** [review]
Thanks Alex! I added some suggestions below with references to the Starter Pack since it's the source of truth for our component API docs. We plan to add some more refined documentation there to make it much easier to consume.

**Pull request merged**